fw-tools/testnet.sh - add credential path, k8s, gateways - no snap

JanneKiiskila opened the pull request:
# Changes

* In case you execute the scripts from some other place than where the script is located, you have to use the full path to the credentials.
* Add k8s, gateways as tested addresses.
* Remove snap as tested address.

[x] pysh-check is clean

# Testing

```
jannek@p520:/ssd/pe-utils$ testnet.sh
[   OK   ]	ping bootstrap.us-east-1.mbedcloud.com
[   OK   ]	ping lwm2m.us-east-1.mbedcloud.com
[   OK   ]	netcat bootstrap.us-east-1.mbedcloud.com 443
[   OK   ]	netcat bootstrap.us-east-1.mbedcloud.com 5684
[   OK   ]	netcat lwm2m.us-east-1.mbedcloud.com 443
[   OK   ]	netcat lwm2m.us-east-1.mbedcloud.com 5684
[   OK   ]	netcat k8s.us-east-1.mbedcloud.com 443
[   OK   ]	netcat gateways.us-east-1.mbedcloud.com 443
[   OK   ]	TLS to bootstrap server (port 5684)
[   OK   ]	TLS to LwM2M server (port 5684)
[   OK   ]	TLS to bootstrap server (port 443)
[   OK   ]	TLS to LwM2M server (port 443)
[   OK   ]	TLS to k8s server (port 443)
[   OK   ]	TLS to gateway server (port 443)
```

Commit:
fw-tools/testnet.sh - add credential path handling

In case you execute the scripts from some other place than where
the script is located, you have to use the full path to the
credentials.

Commit:
README.md - Izuma

Not Pelion anymore...

petedyerarm commented on fw-tools/testnet.sh:
```diff
@@ -7,6 +7,8 @@
 #
 # Credential files will not be found otherwise.
 #
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```
```suggestion
SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
```

petedyer11 reviewed (approved):
LGTM

Commit:
fw-tools/testnet.sh - fix comments, check k8/gw

Fix comments from top (no need to remain in same folder anymore).
Check also now the k8s and gateways connection, as the edge cases
will require check - we should verify that connection to those
works as well.

Removed the snap connectivity, as that is only applicable to
snap (and it has it's tooling in its own repo).

JanneKiiskila commented:
Rebased, force pushed (merge conflict with CHANGELOG.md).
